### ISSUE TITLE
Bi-factor models

### ISSUE THREAD

**Issue by jorgesinval** [alishinski/lavaanPlot#15]
Two suggestions related to bi-factor models:
- add option `bi-factor =` if `T` it means that `covs = F`;
- when plotting bi-factor models, force the general factor to be in the opposite side (in relation to the manifest variables) of the specific factors.

**Comment by alishinski** [comment]
I've got a solution for this in the works, I'll let you know when it's ready.

**Comment by jorgesinval** [comment]
I will wait for it.

**Comment by mrivkale** [comment]
Hi! Was this issue ever solved? I am working with a fairly large bifactor model and trying to plot it in forcing the general factor to the other side. Thanks!

**Comment by alishinski** [comment]
No, not yet, the solution for this that I was working on proved surprisingly unwieldy and problematic, and I had to shelve it. Bifactor models are something I use in my research as well, so it's something that I'll still prioritize going forward when I find time to work on the package, but unfortunately that's all I can say right now.

**Comment by sammy-w** [comment]
> No, not yet, the solution for this that I was working on proved surprisingly unwieldy and problematic, and I had to shelve it. Bifactor models are something I use in my research as well, so it's something that I'll still prioritize going forward when I find time to work on the package, but unfortunately that's all I can say right now.

Any progress on this in the meantime? I would like to plot a bi-factor model as well.
